src/main/java/seedu/address/logic/commands/MarkCommand.java

```diff
@@ -0,0 +1,78 @@
+package seedu.address.logic.commands;
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.client.Client;
+import seedu.address.model.procedure.Completion;
+import seedu.address.model.procedure.Procedure;
+
+public class MarkCommand extends Command {
+
+    public static final String COMMAND_WORD = "mark";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Marks the procedure in the client as complete. \n"
+            + "Parameters: CLIENT_INDEX, PROCEDURE_INDEX (must be both positive integers) \n"
+            + "Example: mark 1 1";
+
+    public static final String MESSAGE_SUCCESS = "Procedure successfully marked as complete.";
+
+    public static final String MESSAGE_PROCEDURE_NOT_DISPLAYED = "No procedure list is retrieved yet! You can call "
```

Comment:
Is there a reason why listProc must be called beforehand?

Reply:
should I edit this part? User can only mark the task on display as complete or incomplete.